deltachat-ios/ViewModel/ContactCellViewModel.swift

```diff
@@ -17,7 +18,7 @@ enum CellModel {
 }
 
 struct ContactCellData {
-    let contactId: Int
+    let contact: DcContact
```

Comment:
i was first facing this change very skeptical as that may be easily result in a performance degradation due to additional database calls.

however, for both places, this struct is created, DcContact is created anyway, so it is okay from that point of view.

memory might still be an issue - as well as missing updates (contact is not updated while contactId forces a reload), i did not checked these points in detail yet.

Reply:
i added a commit that discards the additional caching, just to avoid introducing new issues as side effects here. the code is also closer to the old one and easier to review.